pkg/kubelet/config/file.go

```diff
@@ -138,23 +138,23 @@ func (s *sourceFile) extractFromDir(name string) ([]*v1.Pod, error) {
 
 		switch {
 		case statInfo.Mode().IsDir():
-			glog.V(1).Infof("Not recursing into config path %q", path)
+			glog.V(1).Infof("Not recursing into manifest path %q", path)
 		case statInfo.Mode().IsRegular():
 			pod, err := s.extractFromFile(path)
 			if err != nil {
-				glog.V(1).Infof("Can't process config file %q: %v", path, err)
+				glog.V(1).Infof("Can't process manifest file %q: %v", path, err)
 			} else {
 				pods = append(pods, pod)
 			}
 		default:
-			glog.V(1).Infof("Config path %q is not a directory or file: %v", path, statInfo.Mode())
+			glog.V(1).Infof("Manifest path %q is not a directory or file: %v", path, statInfo.Mode())
```

Comment:
At the very least, "can't process manifest file" is error-class, even if we choose to ignore it. That comment only says that we shouldn't ignore the rest of the files just because one was problematic.